### DISCUSSION TITLE
TMC Pro instead of Audi connect

### DISCUSSION THREAD

**Discussion by Putscho24** [Mr-MIBonk/M.I.B._More-Incredible-Bash#302] [Q&A]
Hello,

I have installed MIB on my MHI2Q Unit in my Audi Q5. Everything works perfect and I am happy so far. Just one question:

Can I activate TMC Pro somewhere in MIB/Green Menu? I had TMC Pro in the past in a Mercedes B Class and the live traffic data was very good. So I thought maybe it is possible also in my AUDI now.

Thank you in advance for your help.

**Comment by harman-f** [comment]
Yes, there is a function.

**Reply by Putscho24** [reply]
Thank you for your reply. That sounds good. Can you tell me how to activate that it will work in Germany?

**Reply by i3oricua** [reply]
Hello. Was/is there is a response to this that someone could share?
